### ISSUE TITLE
NVDA Issue with asp.net Listbox - Unable to traverse options using browse mode

### ISSUE THREAD

**Issue by MohammedAhmed679664** [MohammedAhmed679664/Accessibility#2]
Hi All,

We are using asp.net List box in one of our application and facing below issue in NVDA browse mode.

Issue: When we navigate to List box control and change to Focus mode for items selection, unable to select items by using UP and DOWN arrow keys

Found same issue in below site also. 
https://whatsock.com/tsg/Coding%20Arena/ARIA%20Listboxes/Multiselect/demo.htm

Could anyone help us to know is it a NVDA issue with respect to asp.net list box Issue.

Thanks In Advance
Ahmed
